style.css

```diff
@@ -90,3 +95,226 @@ header {
   padding: 8px;
   list-style-type: none;
 }
+
+.works {
```

Comment:
- [ ] [REQUIRED] You are expected to use flexbox to align the contents of both **_work_** and **_about_** sections.